modules/README.md

```diff
@@ -12,4 +12,4 @@ Modular templates support cross sections of Sysdig Secure feature sets. Each tem
 
 Organizations are supported by setting the following template parameters
 * `IsOrganizational=true`
-* `OrganizationalUnitsIDs=ou-...`
\ No newline at end of file
+* `OrganizationalUnitsIDs=ou-...`
```

Comment:
nit: not quite sure what the line change is here, but should this be `OrganizationalUnitIDs`?

Reply:
great catch. updated.